README.md

```diff
@@ -47,6 +47,86 @@ start = @<language>
 }
 ```
 
+Use `-j` or `--shexj` to generate ShEx JSON Syntax (ShExJ) instead of default ShEx Compact syntax (ShExC).
+
+```
+$ ./shexstatements.sh --shexj examples/language.csv 
```

Comment:
I would suggest moving the JSON output to a separate file and give the link to README (for reducing the size of the main file).